transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/internal/util/Util.java

```diff
@@ -681,42 +676,35 @@ private static void updateIndexes(Model model,
 
     public static void updateMavenDependencies(final List<Dependency> dependencies,
```

Comment:
If you have a dependency to the camel editor you could also use 

https://github.com/fusesource/fuseide/blob/master/editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/utils/MavenUtils.java

We could also move that MavenUtils class to foundation plugin if that helps solving circular dependencies.